changelog/v2.2.0/changes/2417.md

```diff
@@ -4,5 +4,4 @@ operator are now plugins, which makes them easily extensible. The syntax of
 `summarize` now supports specification of output field names, similar to SQL's
 `AS` in `SELECT f(x) AS name`.
 
-The `count` pipeline operator no longer exists. If you relied on the
-undocumented operator's behavior, please let us know.
+The `count` pipeline operator no longer exists.
```

Comment:
I would want to add the "undocumented" back here.

Reply:
The sentence didn't provide any value in my eyes. I think this could be fixed by adding a single word: "the *undocumented* `count` pipeline operator ..."